src/utils/accounts.js

```diff
@@ -57,3 +58,31 @@ export function getActiveAccountWalletId(accounts: Accounts = []): string {
 export function checkIfSmartWalletAccount(account: Account): boolean {
   return account.type === ACCOUNT_TYPES.SMART_WALLET;
 }
+
+export function getAccountName(accountType: AccountTypes): string {
+  switch (accountType) {
+    case ACCOUNT_TYPES.SMART_WALLET:
+      return 'Smart Wallet';
+    case ACCOUNT_TYPES.KEY_BASED:
+      return 'Key Based account';
+    default:
+      return '';
+  }
+}
+
+export function getInactiveUserAccounts(accounts: Accounts): UserAccount[] {
+  return accounts
+    .filter(({ isActive }) => !isActive)
+    .map(account => ({
+      isUserAccount: true,
+      ethAddress: getAccountAddress(account),
+      username: getAccountName(account.type),
```

Comment:
Username that could be either `Smart Wallet` or `Key Based account`? Should be renamed to `label` or whatever, but not `username`